base_frame_id: base_link
odom_frame_id: odom
publish_rate: 50

wheel_separation: 0.57  # wheel width: 0.05
wheel_radius: 0.1

wheel_separation_multiplier: 1.0
left_wheel_radius_multiplier: -1.0
right_wheel_radius_multiplier: 1.0
